.npmrc

```diff
@@ -1 +1,2 @@
-save-exact=true
\ No newline at end of file
+save-exact=true
+node-linker=hoisted
```

Comment:
I added this for compatibility with `tsdx`

@ProchaLu check into this for 1h, try to figure out a way to make it work without this `node-linker=hoisted` (I'm also fine with upgrading more of the transitive dependencies like `rollup`, `rollup-plugin-typescript2` etc). if you don't get a solution with this, document it here, including error message (text), link to failing workflow, link to this PR:

- #126

Reply:
Reverted the `node-linker=hoisted` and fixed the dependencies in:

- https://github.com/upleveled/preflight/pull/403